tutorials/dataframe/df106_HiggsToFourLeptons.C

```diff
@@ -0,0 +1,281 @@
+         // Apply an MC correction for the ZZ decay due to missing gg->ZZ process
+         .DefinePerSample("scale",
+                          [](unsigned int slot, const ROOT::RDF::RSampleInfo &id) {
+                             if (id.Contains("mc_363490.llll.4lep.root")) {
+                                return 1.3f;
+                             } else {
+                                return 1.0f;
+                             }
+                          })
+         // Select electron or muon trigger
+         .Filter("trigE || trigM")
+         // Select events with exactly four good leptons conserving charge and lepton numbers
+         // Note that all collections are RVecs and good_lep is the mask for the good leptons.
+         // The lepton types are PDG numbers and set to 11 or 13 for an electron or muon
+         // irrespective of the charge.
+         .Define("good_lep",
+                 "abs(lep_eta) < 2.5 && lep_pt > 5000 && lep_ptcone30 / lep_pt < 0.3 && lep_etcone20 / lep_pt < 0.3")
+         .Filter("Sum(good_lep) == 4")
+         .Filter("Sum(lep_charge[good_lep]) == 0")
+         .Define("goodlep_sumtypes", "Sum(lep_type[good_lep])")
+         .Filter("goodlep_sumtypes == 44 || goodlep_sumtypes == 52 || goodlep_sumtypes == 48")
+         // Apply additional cuts depending on lepton flavour
+         .Filter(
+            "GoodElectronsAndMuons(lep_type[good_lep], lep_pt[good_lep], lep_eta[good_lep], lep_phi[good_lep], "
+            "lep_E[good_lep], lep_trackd0pvunbiased[good_lep], lep_tracksigd0pvunbiased[good_lep], lep_z0[good_lep])")
+         // Create new columns with the kinematics of good leptons
+         .Define("goodlep_pt", "lep_pt[good_lep]")
+         .Define("goodlep_eta", "lep_eta[good_lep]")
+         .Define("goodlep_phi", "lep_phi[good_lep]")
+         .Define("goodlep_E", "lep_E[good_lep]")
+         .Define("goodlep_type", "lep_type[good_lep]")
+         // Select leptons with high transverse momentum
+         .Filter("goodlep_pt[0] > 25000 && goodlep_pt[1] > 15000 && goodlep_pt[2] > 10000")
+         // Compute invariant mass
+         .Define("m4l", "ComputeInvariantMass(goodlep_pt, goodlep_eta, goodlep_phi, goodlep_E)")
+         // Reweighting of the samples is different for "data" and "MC"
+         .DefinePerSample("reweighting",
+                          [](unsigned int slot, const ROOT::RDF::RSampleInfo &id) { return id.Contains("mc"); });
+
+   // Define the weight column (scale factor) for the MC samples
+   auto df_mc = df_analysis.Filter("reweighting == true")
+                   .Define("weight", ("scaleFactor_ELE * scaleFactor_MUON * scaleFactor_LepTRIGGER * "
+                                      "scaleFactor_PILEUP * mcWeight * scale * xsecs / sumws * lumi"));
+
+   // Book histograms for individual MC samples
+   auto df_higgs = df_mc.Filter("sample_category == \"higgs\"")
```

Comment:
```suggestion
   auto df_higgs = df_mc.Filter(R"(sample_category == "higgs")")
```

This is again a matter of taste. You could also use [raw string literals](https://stackoverflow.com/questions/56710024/what-is-a-raw-string#56710079) to avoid having to use the escape sequences.